.github/workflows/bullseye-coverage.yml

```diff
@@ -201,15 +201,15 @@ jobs:
     env:
       CONFIG_POWER_ONLY: false
       PRAGMA_SUFFIX: -vm
-      OPERATIONS_EMAIL: brian.murrell@intel.com
+      OPERATIONS_EMAIL: core-daos-devops@groups.int.hpe.com
```

Comment:
We can't use core-daos-devops@groups.int.hpe.com here as this is a public repo and the groups.int.hpe.com will only deliver e-mail originating inside of HPE.

Reply:
Normally that environment variable should be set by Jenkins.

For github actions, there should be a way for the github action to notify the submitter.   We don't want e-mail sent from actions that are run by non HPE employees.

Reply:
With GHA it shouldn't be necessary to provide an email since if a GHA fails, the user will get a notification by default. But that might require adjusting code paths to propagate errors.

Reply:
I have revert all e-mail related changes.
Let's fix them when we know what address, if any, shall be used.